subprojects/tooling-api/src/crossVersionTest/groovy/org/gradle/integtests/tooling/r22/ClientShutdownCrossVersionSpec.groovy

```diff
@@ -49,7 +54,7 @@ class ClientShutdownCrossVersionSpec extends ToolingApiSpecification {
 
     def "cleans up idle daemons when tooling API session is shutdown"() {
         withConnection { connection ->
-            connection.model(GradleBuild).setJvmArguments(JVM_OPTS).get()
+            connection.model(GradleBuild).setJvmArguments(getBuildJvmArguments()).get()
```

Comment:
You could reference this via a Groovy property `buildJvmArguments`, right?

Reply:
done